utils/adoption_forecast_parsing.py

```diff
@@ -4,23 +4,98 @@
 
 from io import BytesIO
 import json
+from pathlib import Path
+import logging
```

Comment:
Let's be consistent with the other files and use loguru here instead of the logging library.

```suggestion
from loguru import logger
```